src/server.cpp

```diff
@@ -831,16 +845,23 @@ static CTimingMeas JitterMeas ( 1000, "test2.dat" ); JitterMeas.Measure(); // TE
                 const int iStartChanCnt = iBlockCnt * iMTBlockSize;
                 const int iStopChanCnt  = std::min ( ( iBlockCnt + 1 ) * iMTBlockSize - 1, iNumClients - 1 );
 
-                FutureSynchronizer.addFuture ( QtConcurrent::run ( this,
-                                                                   &CServer::DecodeReceiveDataBlocks,
-                                                                   iStartChanCnt,
-                                                                   iStopChanCnt,
-                                                                   iNumClients ) );
+                Futures.push_back ( tpThreadPool->enqueue ( CServer::DecodeReceiveDataBlocks, 
+                                                    this, iStartChanCnt, iStopChanCnt, iNumClients ) );
+
+                // FutureSynchronizer.addFuture ( QtConcurrent::run ( &CServer::DecodeReceiveDataBlocks,
```

Comment:
Please delete code no longer used.  Git allows it to be recovered easily, if needed.  Commenting it out makes the code unclear.